### PR TITLE
Natsuki and horror topic

### LINKED ISSUE #69: [ FEATURE REQUEST ] - Natsuki and horror
# Overview:
This feature will introduce a new topic prompted by Natsuki, where she talks about exactly why she dislikes the horror genre.

## Must haves:
- New topic "Thoughts on horror"; where Natsuki will elaborate on why she dislikes the horror genre and notes that while she can appreciate the creativity and effort that goes into those works, they aren't for her.

## Could haves:
N/A

## Is this feature request related to a problem? Please describe.
This is a new feature.

### PR DESCRIPTION
**Pull request by Blizzardsev**
This will introduce the following:

- One new topic, `talk_thoughts_on_horror`.

Closes #69

### CODE REVIEW

File: game/script-topics.rpy [hunk 1]
```diff
@@ -1883,6 +1883,79 @@ label talk_work_life_balance:
...
+        n "You know, I don't think I explained why I dislike horror."
+        n "I get everyone has their tastes, but it isn't for me."
+
+    elif jn_affinity.get_affinity_state() <= jn_affinity.RUINED:
+        n "...I was about to share some of my thoughts on horror with you."
+        n "Or at least,{w=0.1} I was thinking about it."
+        n "...But then do you know what I realized,{w=0.1} [player]?"
+        n "I hate horror -{w=0.1} not that you'd care -{w=0.1} and honestly?"
+        n "...I'm starting to think being here with you is horror enough.{w=0.2} Heh."
+        return
```
Comment (spaghetsie) [inline game/script-topics.rpy:1921]: I think these are meant to be swapped. Unless I'm just dumb, which is quite likely in fact, the second and third `if` statements will never be called, no?

**Review by spaghetsie — CHANGES_REQUESTED** [review]
Just a small affinity logic thing.

File: game/script-topics.rpy (hunk 1 above)
Comment (Blizzardsev) [reply game/script-topics.rpy]: You're right, was basically big dum and messed up my conditioning - corrected in next commit

**Review by spaghetsie — APPROVED** [review]
all good

**Pull request merged**